sass/stylesheets/signin.sass

```diff
@@ -28,3 +28,73 @@ body
       margin-bottom: 10px
       border-top-left-radius: 0
       border-top-right-radius: 0
+
```

Comment:
Yes but main never gets called for the signin page - we can change that if you want, but the last i remember we were trying to keep different templates separate?  correct me if i'm wrong. :)  But yes, all this button code is exactly the same